doc/manual/source/store/drv/building.md

```diff
@@ -0,0 +1,88 @@
+# Building
+
+TODO
+
+## Normalizing derivation inputs
+
+TODO
+
+Each input must be [realised] prior to building the derivation in question.
+At that point, the derivation can be *normalized*, by replacing each input deriving path with its store path --- which we now know since we've realised it.
+
+## Builder Execution
```

Comment:
This section is taken from the language derivation docs with no modification but link updates. It is moved here.